scrapper.py

```diff
@@ -44,18 +75,50 @@ def get_search_urls(self):
         pass
 
 
+class ArticleParser:
+    def __init__(self, full_url, identification):
+        self.article_url = full_url
+        self.article_id = identification
+        self.article = ''
+
+
 def prepare_environment(base_path):
     """
     Creates ASSETS_PATH folder if not created and removes existing folder
     """
-    pass
+    try:
+        os.rmdir(base_path)
+    except FileNotFoundError:
+        os.mkdir(base_path)
 
 
 def validate_config(crawler_path):
     """
     Validates given config
     """
-    pass
+    with open(crawler_path) as file:
+        config = json.load(file)
+
+    seed_urls = config["seed_urls"]
```

Comment:
are you sure those keys will be every time?

Reply:
done